app/controllers/users_controller.rb

```diff
@@ -168,7 +168,7 @@ def set_vars_from_curr_user
   
 	def show
 		set_vars_from_curr_user
-		if current_user.recipes.blank?
+		if current_user.recipes.where(:type => "PlanRecipe").blank?
```

Comment:
This makes it clear that you’re Distinguishing planrecipes from favoritedrecipes, I like it. Nice coding style